aiofreepybox/aiofreepybox.py

```diff
@@ -54,9 +54,13 @@ def __init__(self, app_desc=app_desc, token_file=token_file, api_version='v3', t
         if not self._is_app_desc_valid(self.app_desc):
             raise InvalidTokenError('invalid application descriptor')
 
-        cert_path = os.path.join(os.path.dirname(__file__), 'freebox_root_ca.pem')
+        if host.endswith('.freeboxos.fr'):
```

Comment:
On second thought, I'm thinking about reversing the check to only use the `freebox_root_ca.pem` CA for `fbox.fr` subdomains.

What do you think?

Reply:
I was thinking the same thing, it is possible to configure a fully custom domain name that does not necessarily end in `freeboxos.fr`. I even wonder if testing domains is really a good idea, perhaps it would be better to simply register the Free certificates as trusted authorities and always use the default SSL context creation. I'm not sure exactly yet how to do that, I'll do some tests.

By the way, I didn't manage to reproduce issue #2 at home. What Freebox do you have? If I continue failing to reproduce, would you be willing to perform some tests if I push a PR?

Reply:
I'm not not fond of trusting custom CAs  as you lose all the security work done upsream by the distribution.

I have a Freebox v6 from december 2010, current firmware version 3.5.2.

Open for testing.

Reply:
Hmm, I'm not sure I understand: the CA would only be trusted in the context of the library, what would be the risk? (sorry if that's a dumb question, I'm pretty much discovering SSL as we're speaking!)

If trusting the Freebox CA is a security risk, perhaps we could make it optional so that the user can decide whether she wants to enable insecure access or not, instead of relying on the domain used. I guess this parameter would need to be propagated to Home Assistant as well, but it would make the setup more complex, and prevent automatic discovery to work correctly.

Reply:
Let's continue this discussion on PR #3